example/counter/counter.go

```diff
@@ -72,5 +72,14 @@ func (app *CounterApplication) Commit() types.Result {
 }
 
 func (app *CounterApplication) Query(query []byte) types.Result {
-	return types.NewResultOK(nil, Fmt("Query is not supported"))
+	queryStr := string(query[:])
+
+	switch queryStr {
```

Comment:
let's just use `hash` and `tx`, no need for `Count`. Also, let's just  return the value, no need for the extra english.

Finally, seems like `ErrUnknownRequest` might be more appropriate